src/InputBar.js

```diff
@@ -69,17 +70,17 @@ export default class InputBar extends React.Component {
                             <button class="btn btn-primary" type="submit"
                                 title="Roll"
                                 disabled={!formulaIsRollable(this.props.formula)}
-                                onClick={this.handleRollClick}>
+                                onClick={this.handleRollClick}><span className="sr-only">Submit</span>
```

Comment:
Not critical, but shouldn't the text match the button title "Roll"?

Reply:
yes, I actually didn't even notice that. Typically we have not typically been using this attribute based on the [accessability concens](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#Accessibility_concerns) so I don't even think i noticed this. 

Also makes me think maybe the text is worth while to show all users.

changes have been pushed.